website/docs/plugin/sdkv2/logging/http-transport.mdx

```diff
@@ -0,0 +1,211 @@
+---
+page_title: Plugin Development - Logging HTTP Transport
+description: |-
+  SDKv2 provides a helper to easily send all the HTTP Transactions to structured logging.
+---
+
+# Background
+
+Since Terraform `v0.9.5`, the public interface used by Provider Developers included an _helper_ inside the package `helper/logging`: [`NewTransport()`](https://github.com/hashicorp/terraform-plugin-sdk/blob/main/helper/logging/transport.go), an implementation of the Golang standard library [`http.RoundTripper`](https://pkg.go.dev/net/http#RoundTripper). This was provided as an easy way for Terraform provider developers to add _DEBUG-level_ logging to the HTTP transactions of their provider.
+
+Unfortunately this helper is designed to log the entirety of each request/response it handles, including every header and the entire body, regardless of how _sensitive_ the content is. Over time security concerns have been [raised](https://github.com/hashicorp/terraform-plugin-sdk/issues/2), and required provider developers to build [workarounds](https://github.com/hashicorp/terraform-provider-tfe/pull/479).
+
+The recommended way to produce logs from your provider is using **[terraform-plugin-log](https://www.terraform.io/plugin/log)** library, and that provides [log filtering](https://www.terraform.io/plugin/log/filtering) functionality. So, we built a new _RoundTripper_ to deprecate the existing one.
+
+
+# Setting up logging for HTTP Transactions
+
+SDK offers helpers to put in place logging of HTTP Transactions: it’s built on top of **[terraform-plugin-log](https://www.terraform.io/plugin/log)**, so it allows to leverage its features, without having to write the whole implementation of `http.RoundTripper`.
+
+There are 2 functions inside the package `helper/logging`, each targeting a specific logging setup that the provider developer might have chosen (see [“Writing Log Output”](https://www.terraform.io/plugin/log/writing) for details):
+
+* `NewLoggingHTTPTransport(transport http.RoundTripper)`
+* `NewSubsystemLoggingHTTPTransport(subsystem string, transport http.RoundTripper)`
+
+Using one or the other depends on the developer, and if they want their logging to happen against the `tflog` Provider root logger, or against a `tflog` Provider [Subsystem logger](https://www.terraform.io/plugin/log/writing#subsystems). For the latter, the only thing to remember is that the `subsystem` string used with `NewSubsystemLoggingHTTPTransport()`, has to match the [pre-created subsystem logger name](https://www.terraform.io/plugin/log/writing#create-subsystems).
+
+To make use of our logging transport, there are 2 things that a provider developer should do:
+
+* Setting up the HTTP Client to use this new Transport
+* Ensuring that each HTTP Request, before being submitted to the Client, carries the configured `context.Context`, storing the desired logging configuration
+
+
+## Creating HTTP Client that uses the new logging transport
```

Comment:
These seem like sub-steps of the higher level setup heading, so I'm making them H3. let me know if I'm incorrect on that! :) 
```suggestion
### Creating the HTTP Client 
```